csrc/codegen.cpp

```diff
@@ -1158,6 +1158,24 @@ class CudaKernelGenerator : private OptOutConstDispatch {
 
   void handle(const LoadStoreOp* ldst) final {
     auto optype = ldst->opType();
+
+    // Note: segmenter load is just a WAR to allow segmenter to breka kernel
+    if (optype == LoadStoreOpType::SegmenterLoad) {
```

Comment:
I think we should convert this to the normal `Set` at the beginning of lowering as it has no meaning once segmentation is done.

Reply:
Let me know if you would like me to do this.